### PR TITLE
Finished create users feature

### PR DESCRIPTION
**Pull request by TerryvilleDev**

### CODE REVIEW

File: TAInformation/Models/validator_methods.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
+from TAInformation.Models.base_user import BaseUser
```
Comment (bptome) [inline TAInformation/Models/validator_methods.py:1]: Good idea for the new file

File: TAInformation/views.py [hunk 2]
```diff
@@ -105,14 +105,96 @@ def post(self, request):
...
+                    request.session['email'],
+                    "",
+                    ""
+                )
+            case _:
+                current_user: BaseUser
+
+        result_dict = {}
+        the_id = -1 if request.POST.get('user_id') == "" else int(request.POST.get('user_id'))
+        match int(request.POST.get('role')):
```
Comment (bptome) [inline TAInformation/views.py:162]: What's the purpose of instantiating the class object, just to save its data into the database? Is it possible to just get the data from the form and put it into the database? Then you don't even need to use any case statements.

**Review by bptome — APPROVED** [review]
I'll approve it since working software is more important, but I think that your view class can be simplified. I think that all you need to do on that page is get the data from the form, (validate it?), and then save it to the database. ie: User(request.session['user_id']   ...... ).save()

Also, I'm not sure we want the Admin user to be able to specify user ids since they may accidentally overwrite a user with that existing id

File: TAInformation/views.py (hunk 2 above)
Comment (TerryvilleDev) [reply TAInformation/views.py:163]: I could try using a User model object instead and change the create_user and validators to work for a model instead.

**Comment by TerryvilleDev** [comment]
> Also, I'm not sure we want the Admin user to be able to specify user ids when creating a new user since they may accidentally overwrite a user with that existing id

This won't happen as create_user checks to see if an user with the specified id exists before saving the new user.

**Review by OmarHHammad — APPROVED** [review]
LGTM

**Pull request merged**